### PR TITLE
Create branch form journal-week-3

### PR DESCRIPTION
**Pull request by ShayHall**
This week, you should produce not one but two journal entries. No worries, because you've got all those ideas backlogged in your parking-lot.md, right? (Uh, right?) Yes, but to get two entries out in a week, you'll either have to learn to write faster or learn to write more efficiently. Maybe you need to spend less time drafting or more time planning? Perhaps you should time-box yourself and just release whatever you've written at the end of the timebox? You could always just forget about proofreading and editing... just write!

All of the topics from the last three weeks are still fair game. Write one reflective post -- review an event, assignment, or lecture from the last three weeks that includes lessons learned or bright/dim/blocker -- AND one tutorial-style post -- explain one small thing to someone who's never seen it before: a tool, trick, or technique you've learned how to use. Include code!

If you can't think of what to name your Reflective Journal, journal-week-3.md will suffice. Similarly, tutorial-week-3.md is a generic name you can use until you have a better one. Start with a quick "flag in the sand" for both: create a file and dump some rough ideas in it.

### CODE REVIEW

**Comment by s4dmw** [comment]
good starting points for journal and tutorial!

I struggle quite a bit with time management so i always like to see how others handle it...

:+1:

**Comment by jeffworley** [comment]
Shay, this looks like a solid outline!  I look forward to seeing if you can match your extremely detailed journals from last week on these oh so pertinent and frustrating topics :smile: Keep up the good work :cool: guy. :+1:

**Comment by rosalsm** [comment]
Flag in the sand, done! +1:

File: tutorial-week-3.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# CSS is like Calvin Ball
```
Comment (kellymurray) [inline tutorial-week-3.md]: this is a huge topic - you might want to choose one aspect to instruct on. Make them actionable items! :)

**Comment by drumslayert** [comment]
Nice work Shay! You write so much better than me I have a hard time finding problems with your work. Keep up the great work.

**Pull request merged**